addons/web/static/src/search/search_bar_menu/search_bar_menu.js

```diff
@@ -118,9 +118,9 @@ export class SearchBarMenu extends Component {
      * @returns {boolean}
      */
     validateField(fieldName, field) {
-        const { sortable, store, type } = field;
+        const { groupable, type } = field;
         return (
-            (type === "many2many" ? store : sortable) &&
+            groupable &&
             fieldName !== "id" &&
             GROUPABLE_TYPES.includes(type)
```

Comment:
isn't the check on `groupable` enough? Is `id` ever groupable? And do we have fields not in `GROUPABLE_TYPES` that can be `groupable` ?

Reply:
In the backend point of view, every type of field is groupable (if they are a valid sql representation). But for the client, it makes much more sense to be constraint with `GROUPABLE_TYPES`.